### PR TITLE
updating to go modules, compiles cleanly on go 1.16 on Mac M1 silicon computers

### PR DESCRIPTION
**Pull request by madsenwattiq**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module wattiq.io/assume-role
+
+go 1.15
```
Comment (czerasz) [inline go.mod:3]:
```suggestion
go 1.16
```

**Comment by czerasz** [comment]
seems that similar stuff was already suggested https://github.com/remind101/assume-role/pull/44/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6 in the past

**Review by LoveriusB — APPROVED** [review]

**Comment by kave** [comment]
any update on when this will be merged?

**Comment by madsenwattiq** [comment]
> any update on when this will be merged?

None.  PR #58 is a more complete version of this one -- I think we should all follow what @anherrera did there because it additionally updates dependencies for recent AWS SDK.  I'm going to delete this PR and work with hers instead.

**Pull request closed without merging**